reload/dpl-platform-poc
# Adding ingress and cert manager

#### What does this PR do?
Adding ingress and cert manager

#### How should this be manually tested?
Run `dplsh` and `cd infrastructure/terraform/alpha/poc02/tasks/`
Run `task setup-ingress` and verify that ingress and cert-manager is initialized in cluster.
